src/Microsoft.Framework.Caching.SqlServer/SqlServerCacheOptions.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Framework.Internal;
+using Microsoft.Framework.OptionsModel;
+
+namespace Microsoft.Framework.Caching.SqlServer
+{
+    public class SqlServerCacheOptions : IOptions<SqlServerCacheOptions>
+    {
+        public ISystemClock SystemClock { get; set; } = new SystemClock();
+
+        public TimeSpan ExpirationScanFrequency { get; set; } = TimeSpan.FromMinutes(10);
```

Comment:
Naming: when I read frequency I think about something that the higher the value the more often it happens. Should this be interval or something like that?